.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Bug report
+description: Create a report to help us improve (search for duplicates first)
+title: '[BUG]: '
+labels: bug
+assignees: []
+body:
+
+ - type: checkboxes
+   id: operating-system
+   validations:
+     required: true
+   attributes:
+    label: Operating System
+    description: Which operating system have you used and triggered the issue?
+    options:
+      - label: macOS
+      - label: Microsoft Windows
+      - label: Linux
+      - label: Unix
+      - label: Other (Tell us at the Specifications part!)
```

Comment:
Avoid use ! to not come off aggressive, also remove Unix, it should go under other, default should be Windows (without Microsoft)